docs/user/faq/general.md

```diff
@@ -1,133 +1,185 @@
-
-- If you want to use OpenVPN package, you probably will need to open the ports: TCP port 8092 and 1194 UDP
-- In the case of Wireguard package, it could be you need to open the port 51820 UDP.
-
-In the mod advanced configuration, if you want to set up ssh access to your dappnode, you will have to open the ssh port in the router to be able to access it from the outside of the network.
-
-## What happens if my node is offline while validating - do I get slashed?
-
-There are two kinds of punishment when you validate on Ethereum 2.0 (this is applied in gnosis chain too):
-
-- Penalty: Reduce the token of the validator
-- Slashing: Remove the validator from the beacon-chain and the balance of the validator continues decreasing.
-
-The penalty happens when a validator is not active or misses some attestation.
-
-A validator gets slashed if :
-
-- Acts like a proposer and signs two different beacon blocks for the same slot
-- Acts like an attester and signs two different attestations with the same target
-
-  The second one can happen in case you have two validators running with the same key.
+<details>
+  <summary>What is the purpose of Dappnode?</summary>
+  <div>
+    <div>The main purpose or mission of Dappnode is to make it as easy as possible for everyone, even those with limited technical skills, to participate in decentralized networks and the burgeoning Web3 ecosystem. Furthermore, Dappnode aims to increase the decentralization of blockchains and other Web3 infrastructure by lowering the technical barriers of entry to this growing and exciting ecosystem.</div>
+    <br/>
+  </div>
+</details>
+
+<details>
+  <summary>What can I do with Dappnode?</summary>
+  <div>
+    <div>The Dappnode platform lets you run nodes and applications interfacing with all different kinds of Web3 projects including blockchains like Bitcoin, Ethereum, Gnosis Chain and many more in an easy and user-friendly way. 
+    <br/> <br/>
+    Furthermore, you can earn passive income by running validators securing a blockchain or staking nodes in networks like Ethereum, Gnosis Chain, Avalanche and more to come.  Other Web3 projects you can currently participate in to generate passive income with your Dappnode is to share disk space on the StorJ network, or to share some of your bandwidth for the decentralized VPN provider Mysterium.  Our core developers and community of builders are constantly adding integrations for other Web3 projects, many with economic incentives.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>What hardware do I need to run a Dappnode?</summary>
+  <div>
+    <div>
+  You can run the Dappnode platform on various machines ranging from low-end hardware (such as old laptops) to high-end hardware (such as high performance servers).
```

Comment:
old docs used to specifically say never on a laptop because if it gets closed its off, and we should not reccomend using old hardware anymore, requirements are rising quickly and the merge exposed a lot of deficencies mainly due to us not updating the recommended and minimum specs and many just doing the minimum thinking its ok and then asking the devs why they cant start dappnode on a 5 year old NUC with 8gb RAM and an HDD